deploy/role.yaml

```diff
@@ -314,4 +314,13 @@ rules:
       - update
       - delete
       
-  # END Permissions needed for our namespaces, but not given by "admin" role
\ No newline at end of file
+  # END Permissions needed for our namespaces, but not given by "admin" role
+
+  # Permission to fetch identity to get email for created Keycloak users in openshift realm
+  - apiGroups:
+      - user.openshift.io
+    resources:
+      - identities
+    verbs:
+      - get
```

Comment:
one question do we also need list on this?

Reply:
It wasn't required when I was testing it anyway :thinking: . From what I'm aware of, the operator only gets this resource specically here and wasn't listed:
https://github.com/integr8ly/integreatly-operator/blob/ca9144506bb8108423a1a800c1a3670d6d8207cc/pkg/resources/user/userHelper.go#L27-L28

Reply:
ok great 👍